packages/serve/test/auth/cannerPATAuthenticator.spec.ts

```diff
@@ -0,0 +1,237 @@
+import * as sinon from 'ts-sinon';
+import { IncomingHttpHeaders } from 'http';
+import { Request, BaseResponse } from 'koa';
+import { CannerPATAuthenticator } from '@vulcan-sql/serve/auth';
+import { AuthResult, AuthStatus, KoaContext } from '@vulcan-sql/serve/models';
+
+const wrappedAuthCredential = async (
+  ctx: KoaContext,
+  options: any,
+  resolveValue: any = null
+): Promise<AuthResult> => {
+  const authenticator = new CannerPATAuthenticator({ options }, '');
+  if (resolveValue) {
+    sinon.default
+      .stub(authenticator, <any>'fetchCannerUser')
+      .resolves(resolveValue);
+  }
+  await authenticator.activate();
+  return await authenticator.authCredential(ctx);
+};
+
+const getTokenInfo = async (
+  ctx: KoaContext,
+  options: any
+): Promise<Record<string, any>> => {
+  const authenticator = new CannerPATAuthenticator({ options }, '');
+  await authenticator.activate();
+  return await authenticator.getTokenInfo(ctx);
```

Comment:
Same suggestion as above creating the authenticator and calling activate method with the target method at the same place.

Reply:
I've removed this function because it is not used after refactoring.